packages/itinerary-body/src/TransitLegBody/index.js

```diff
@@ -158,6 +163,7 @@ export default class TransitLegBody extends Component {
 TransitLegBody.propTypes = {
   config: configType.isRequired,
   leg: legType.isRequired,
+  LegIcon: PropTypes.elementType.isRequired,
```

Comment:
Shouldn't this be optional (with a default value provided)?

Reply:
If we do that, there should be a package-wide default value. That should be handled separately IMO.